src/routes/solid-start/building-your-application/rendering-modes.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: "Rendering Modes"
+---
+
+SolidStart has 3 kinds of rendering modes: `sync`, `async`, and `stream`.
+Let's talk about how each of them work and which one to pick.
```

Comment:
This line makes sense in a blogpost or guide. Maybe for this scenario you can just remove it?